testing/integration/install_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build integration
+
+package integration
+
+import (
+	"context"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	atesting "github.com/elastic/elastic-agent/pkg/testing"
+	"github.com/elastic/elastic-agent/pkg/testing/define"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestInstall(t *testing.T) {
+	info := define.Require(t, define.Requirements{
```

Comment:
I understand that we need to call `Require` as first function in the test, however it's not clear to me if we have to do something with the `info` object returned (seems unused)

Reply:
Thanks for catching that.  I was using `info` but not anymore.  Will remove.

Reply:
Removed.